pages/api/check-status.ts

```diff
@@ -81,24 +81,36 @@ export default async function handler(
   req: NextApiRequest,
   res: NextApiResponse<CheckStatusReponse | string>
 ) {
-  if (req.method !== 'POST') {
-    return res.status(405).send(`Invalid request method ${req.method}`)
+  if (req.method !== 'GET') {
+    res.status(405).send(`Invalid request method ${req.method}`)
+    return
```

Comment:
Why break this into two lines?

Reply:
Because you are not suposed to return the response but mutatate it with the handler function. So I mutate the reponse then early return.

Reply:
ahh, my eyes have lied to me - I was sure I saw returns everywhere in the examples, but they aren't there at all...